### PR TITLE
chore: result page

### PR DESCRIPTION
**Pull request by izabellalarsson**
- Updated icons
- New design to "testa en annan sajt"
- Welcome msg 

Screenshot 2022-01-20 at 09 51 53

### CODE REVIEW

File: packages/web/src/tests.js [hunk 1]
```diff
@@ -78,10 +78,11 @@ async function showTest(ctx) {
     const test = await connection.one(getTestRunByID(id))
     const result = await connection.any(getTestResultByID(id))
     if (result.length === 0) { return ctx.render('tests/loading') }
-
+    const failedTestResults = result.filter(a => !a.test_output.passed)
+    const allTestsPassed = failedTestResults.length ? false : true
```
Comment (mull) [inline packages/web/src/tests.js:82]:
```suggestion
    const allTestsPassed = result.some(t => !t.test_output.passed)
```

**Review by mull — COMMENTED** [review]
Should the thumbs up animation keep repeating?

File: packages/web/src/tests.js (hunk 1 above)
Comment (izabellalarsson) [reply packages/web/src/tests.js:82]: this always returns true 🤔

**Comment by izabellalarsson** [comment]
> Should the thumbs up animation keep repeating?

I think so, but I can dubble check it

File: packages/web/src/tests.js (hunk 1 above)
Comment (mull) [reply packages/web/src/tests.js:82]: 😕 it shouldn't :D

Comment (mull) [reply packages/web/src/tests.js:82]: Oh I have it the wrong way around, this would return true if 1 test failed. Add a bang `!` before `result.some`

**Review by mull — APPROVED** [review]

**Pull request merged**